### DISCUSSION TITLE
Some Basic Question about lazy vim

### DISCUSSION THREAD

**Discussion by syedtalaljilani** [LazyVim/LazyVim#2246] [Q&A]
1. how to delete file in lazyvim 
2. how to copy , paste , cut  text in insert mode

**Comment by joseiedo** [comment]
1. You can open Neotree `<leader> + e` and press the "?" key. This will show you the Neotree keymaps. You can also just type `:h neo-tree`. The key to delete a file is "d"

For the second one,  i really recommend you to change between insert/visual/normal mode
I mean, you can try to use `ctrl-o+p` to paste, hold `shift+mouse` to select and `shift+ctrl+c` to copy, and map some command for cutting texts (ex: `vim.keymap.set("i", "xx", "<Esc>EvBdi")` to cut the current word)

But it's way less efficient than just use the visual/normal mode, try it :)
